packages/cozy-client/src/CozyClient.spec.js

```diff
@@ -607,6 +607,30 @@ describe('CozyClient', () => {
     })
   })
 
+  describe('destroy', () => {
+    it('should remove the document from the store', async () => {
+      client.setData(
+        normalizeData({
+          'io.cozy.todos': [TODO_1]
+        })
+      )
+      const doc = { ...TODO_1, label: 'Buy croissants' }
+      client.store.dispatch.mockReset()
```

Comment:
Nit: I find it a bit odd to reset a mock in the middle of a test, I think it's usually preferable to do those in `beforeEach/afterEach`.

Reply:
It's because I am not interested in the dispatch that the setData has done. Would it be clearer for you if I moved the above lines in a beforeEach ?

Reply:
I've made the change to put lines in a beforeEach, I agree that it's clearer.